source/installation/kafka-deployments/migrator-with-kafka.txt

```diff
@@ -0,0 +1,73 @@
+.. _kafka-intro:
+
+======================================
+Relational Migrator with Kafka Connect
+======================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can use Relational Migrator to migrate data to and from Kafka. 
+Relational Migrator can be used as a Kafka Connect plug-in that improves 
+resilience and scalability of big data migration jobs.
+
+Use Cases
+---------
+
+Relational Migrator with Kafka is intended for large and long running jobs. 
+
+Behavior
+--------
+
+Relational Migrator works with Kafka as a Kafka Connect sink connector. A 
+`Debezium connector <https://debezium.io/documentation/reference/stable/connectors/index.html>`__ 
+captures data events from your source database, while Relational Migrator 
+interprets and transforms events from the Debezium source and sends them to a 
+MongoDB cluster that acts as a data sink.
+
```

Comment:
(graphic) 
Your outline states that you'll add a graphic here. I think that would be a really nice way to show the information.

I don't think this needs to be a blocking issue, however. If you'd like, please check with Tom and David to see if it makes sense to treat this as a fast follow update, once the bulk of your work has been completed and the docs are published.

Reply:
I added a potential graphic!